### PR TITLE
dns: Add Google Postmaster Tools verification

### PR DESCRIPTION
**Pull request by abizer**
no clue if this works

### CODE REVIEW

File: etc/zones/db.ocf.berkeley.edu [hunk 1]
```diff
@@ -33,5 +33,6 @@ g IN MX 5 alt2.aspmx.l.google.com.
 g IN MX 10 alt3.aspmx.l.google.com.
 g IN MX 10 alt4.aspmx.l.google.com.
 g IN TXT "google-site-verification=4C50Ut260ZQBs2yuLmrRnO1OHvPGMdqlRgl7vIxC8YA"
+@ IN TXT "google-site-verification=90n5U42IP0UQMEIAVccUIcM7qOmSXkiOihFbfDaIbRg"
```
Comment (dkess) [inline etc/zones/db.ocf.berkeley.edu]: why do the other ones start with `g` but this one starts with `@`

Comment (cg505) [reply etc/zones/db.ocf.berkeley.edu]: the other ones are for g.ocf.berkeley.edu

**Comment by cg505** [comment]
@abizer is this still needed? I don't know what account was used for the postmaster tools setup

**Comment by nikhiljha** [comment]
Can we get it to 1 year without this pr being merged?

**Comment by cg505** [comment]
does anyone have context for this PR because i literally don't know what happened

**Pull request closed without merging**